app/serializers/cache/footnote_serializer.rb

```diff
@@ -26,12 +23,10 @@ def as_json
         goods_nomenclature_ids: goods_nomenclatures.map(&:goods_nomenclature_sid),
         goods_nomenclatures: goods_nomenclatures.map do |goods_nomenclature|
           goods_nomenclature_attributes(goods_nomenclature)
-        end
+        end,
       }
 
-      measures = footnote.measures.select { |measure| has_valid_dates(measure) }
-      extra_large_measures = measures.size >= 1000
-      unless extra_large_measures
+      unless extra_large_measures?
```

Comment:
maybe name this method `large_measures?` (if it does not exist already)

Reply:
I did exactly the same thing and figured it was a step too far since the response schema attribute name would have to stay the same but with a different method